fog/view/server/src/block_tracker.rs

```diff
@@ -506,20 +507,17 @@ mod tests {
         assert_eq!(block_tracker.next_blocks(&[rec1, rec2]), expected_state);
     }
 
-    // highest_fully_processed_block_count behaves as expected
+    // highest_fully_observed_block_count behaves as expected
     #[test_with_logger]
-    fn highest_fully_processed_block_count_all_empty(logger: Logger) {
+    fn highest_fully_observed_block_count_all_empty(logger: Logger) {
         let mut block_tracker = BlockTracker::new(logger.clone());
 
-        assert_eq!(
-            block_tracker.highest_fully_processed_block_count(&[]),
-            (0, None)
-        );
+        assert_eq!(block_tracker.highest_observed_block_count(&[]), (0, None));
     }
 
-    // Check with a key that hasn't yet processed anything.
+    // Check with a key that hasn't yet observed anything.
     #[test_with_logger]
-    fn highest_fully_processed_block_missing_blocks_nothing_processed1(logger: Logger) {
+    fn highest_fully_observed_block_missing_blocks_nothing_observed1(logger: Logger) {
```

Comment:
```suggestion
    fn highest_observed_block_missing_blocks_nothing_observed1(logger: Logger) {
```

Reply:
I changed back to processed so this is moot.